pkg/cover/canonicalizer.go

```diff
@@ -152,26 +165,45 @@ func findModule(pc uint32, moduleKeys []uint32) (moduleIdx int) {
 	return moduleIdx - 1
 }
 
-func (convert *Convert) convertPCs(cov []uint32, sign signal.Serial) {
+func (convert *Convert) convertPCs(cov []uint32, sign signal.Serial) ([]uint32, signal.Serial) {
 	// Convert coverage.
-	for idx, pc := range cov {
-		cov[idx] = convert.convertPC(pc)
+	var retCov []uint32
+	for _, pc := range cov {
+		convPC, err := convert.convertPC(pc)
+		if err != nil {
+			log.Logf(1, "discarding coverage pc: %v", err)
```

Comment:
This could be a lot of output. I'd increase the log level and just print the number of discarded PCs (you can print e.g. the first one explicitly like `discarded 0xabcd due to err (and 12345 other PCs) during conversion`).

Same for all similar `Logf` calls.

Reply:
Done